active/0002-centalized-mgmt-resource-types-provider-versions.md

```diff
@@ -70,25 +70,19 @@ provider az
 provider kubernetes as k8s // uses optional aliasing of the
 ```
 
-The resolution of the identifiers will be determined from inspection of the closest `bicepconfig.json` to the file merged with the builtin configuration. We propose to add a new section `providers` and `implicitProviders`.
+The resolution of the identifiers will be determined from the closest `bicepconfig.json` to the file merged with the builtin configuration. We propose to add a new section `providerImports` and `implicitProviderImports`.
```

Comment:
I think there was a plan to rename `provider` to `extension`. Do we want to update these properties as well. i.e. `extensionImports` and `implicitExtensionImports`?

Reply:
there is a plan for this but that will be handled as a separate change request on the project. For now since the naming change is not yet implemented we are keeping the current convention